event_registration_mass_confirm/wizard/confirm_attendees.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="event_registration_massconfirm_view" model="ir.ui.view">
+        <field name="name">event.registration.massconfirm.form</field>
+        <field name="model">event.registration.massconfirm</field>
+        <field name="arch" type="xml">
+            <form string="Confirm Registrations">
+                <p>
```

Comment:
as the string is not long put everythin on one line w/ the `p` tags (<p>blab bla</p>).